### ISSUE TITLE
issues with using perky Duck with nvda

### ISSUE THREAD

**Issue by mhussaincov** [nvaccess/nvda#9258]
issues encountered when using perky Duck (braille software) with NVDA

### Actual behavior:
nvda does not read grade2 contracted braille as written in perky Duck software
e.g does not recognize numbers (does not recognize the numeric indicator)
does not recognize character whilst typing on perky Duck

### Expected behavior:
nvda should read g2 braille
it should read a sentence after it's been typed
it should recognize braille numbers and other braille symbols

#### NVDA installed/portable/running from source:
installed

#### NVDA version:
NVDA 2018.4.1
#### Windows version:
windows 7 enterprize

#### Name and version of other software in use when reproducing the issue:
Perky Duck software from duxbury

#### Does the issue still occur after restarting your PC?
YES
#### Have you tried any other versions of NVDA?
NO

**Comment by DrSooom** [comment]
@mhussaincov: Please fill out the Str section (any special settings?) and add the version of Perky Duck where this issue occurs. The newest one should be 1.2.0.3 (or so) which was released on July 9, 2018.
CC: @leonardder and @dkager

**Comment by mhussaincov** [comment]
updated issue
:)

**Comment by mhussaincov** [comment]
hi there,
any updates to this?
interested?

On 11/02/2019, mhussaincov <notifications@github.com> wrote:
> updated issue
> :)
>
> --
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/nvaccess/nvda/issues/9258#issuecomment-462411990


-- 
kind regards,
Majid Hussain

**Comment by Living4God1991** [comment]
I just started using Perky Duck for a course from the Hadley Institute for the Blind, and this is definitely still an issue. I've actually been trying to figure out how to fix this for the last hour or so, thinking maybe I had something set up incorrectly. Luckily, I finally did a web search and found this issue. I'm using Perky Duck 12.3, which is the latest version, on Windows 10 Pro Insider version 1903 (OS Build 18932.1000) with NVDA version alpha-17928,9dfbd2ee.

**Comment by Living4God1991** [comment]
I just did a little more work with Perky Duck, and I believe that what's happening is that when punctuation, contractions, or other symbols are typed, NVDA is reading them as if they were written in computer braille, so dot 2 is read as a 1, dot 6 is read as a comma, dots 2,5,6 is read as a 4, etc.

**Comment by Adriani90** [comment]
cc: @Andre9642, @leonardder I think this is yet another braille translation issue? Or maybe NVDA does not recognize the braille symbols with speech at all?
